lib/spack/docs/environments.rst

```diff
@@ -374,16 +373,24 @@ Developing Packages in a Spack Environment
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The ``spack develop`` command allows one to develop Spack packages in an environment.
```

Comment:
An example of `spack develop foo@bar` plus its corresponding

```yaml
spack:
  develop:
    pkg:
      spec: ...
```

would make it easier to follow the next few paragraphs.